### DISCUSSION TITLE
配置文件是否能支持yaml格式

### DISCUSSION THREAD

**Discussion by wanghonglei5181** [bfenetworks/bfe#842] [Ideas（新功能的建议和讨论）]
使用K8s较多的用户对于yaml格式的配置文件比使用json格式更舒服

**Comment by mileszhang2016** [comment]
抱歉，短期内BFE没有支持yaml格式配置文件的计划。
1. 从可读性的角度，并没有感知到yaml明显优于json
2. BFE的配置文件一般由程序生成。9月底即将开源BFE的控制面，后续BFE开源项目的用户就不需要手工来改写BFE的配置文件了。

btw，bfe ingress是支持yaml的。目前虽然已经在github上可以查看，还没有正式发布。预计将于10月份正式发布。
